### DISCUSSION TITLE
I2c - input and pwm extenders

### DISCUSSION THREAD

**Discussion by greenembrace** [gpiozero/gpiozero#998] [Q&A] [answered]
Hi 
I have a need for more pins of both input and PWM. There are a number of I2C cards that could extend these functions (pins) on the RPi. Is there any way to access these with the GPIOzero library?

daniel

**Comment by bennuttall** [comment, marked as answer]
There have been plans to add support for I2C for a long time but we're not actively working on gpiozero currently so it's not something you should wait for. You could get some basic I2C code working and create gpiozero-style classes if you wanted a similar experience.

**Reply by greenembrace** [reply]
Cheers Ben thanks for the update.
